workloads/workload_vars/migration.yml

```diff
@@ -1,4 +1,4 @@
 ---
-mig_operator_deploy_controller: true
-mig_operator_deploy_ui: true
+mig_operator_deploy_controller: "{{ true if ocp_version=='4' else false }}"
```

Comment:
@djwhatle Does this make more sense ?

Reply:
This makes sense to me :+1: sane default.